db/controllers/comments.controller.js

```diff
@@ -24,3 +24,15 @@ exports.postNewComment = (request, response, next) => {
     })
     .catch(next);
 };
+
+exports.deleteCommentById = (request, response, next) => {
+  const commentToDelete = request.params.comment_id;
+  removeComment(commentToDelete)
+    .then((deleteComment) => {
+      if (!deleteComment) {
+        return response.status(404).send({ message: "Not Found" });
+      }
+      response.status(204).send(deleteComment);
```

Comment:
In this case we don't want to send anything back when we delete the comment. We can just do response.status(204).send().